.env

```diff
@@ -1,9 +1,16 @@
+## Image versions ##
 CAMUNDA_CONNECTORS_VERSION=0.17.1
 CAMUNDA_OPTIMIZE_VERSION=3.10.0-alpha5
-CAMUNDA_PLATFORM_VERSION=8.2.0-alpha5
+CAMUNDA_PLATFORM_VERSION=SNAPSHOT
```

Comment:
Will `SNAPSHOT` be the new default version? Or is it only required to test the new authentication feature?